Lua/Lua.sublime-syntax

```diff
@@ -476,34 +491,50 @@ contexts:
       pop: true
 
   string:
+    - include: single-quoted-string
+    - include: double-quoted-string
+    - include: multiline-string
+
+  single-quoted-string:
     - match: \'
       scope: punctuation.definition.string.begin.lua
       set:
-        - meta_include_prototype: false
-        - meta_scope: string.quoted.single.lua
-        - include: string-content
-        - match: \'
-          scope: punctuation.definition.string.end.lua
-          pop: true
+        - single-quoted-string-body
```

Comment:
I'd prefer it to be inline with `set` as this implementation is a multi-set with just a single item.